tests/test_simple.py

```diff
@@ -193,3 +193,14 @@ def test_parameterized_select_named_pg(postgresql):
     q.insert_some_value(postgresql)
 
     assert q.get_all_values(postgresql, a=1) == [(1, 2, 3)]
+
+
+def test_without_trailing_semi_colon_pg(postgresql):
```

Comment:
I think you can drop the postgresql fixture which is useless since we don't run any query.

```suggestion
def test_without_trailing_semi_colon_pg():
```